### PR TITLE
Additional psi4 attributes for v2

### PR DESCRIPTION
**Pull request by amandadumi**
This removes some print statements and expands psi4 coverage to the missing attributes. Cleans up some print unneeded statements. 

Marking this as WIP so i can work on other attributes and rebase against #1424 once merged.

### CODE REVIEW

File: cclib/attribute_parsers/mosyms.py [hunk 1]
```diff
@@ -186,7 +186,7 @@ def qchem(file_handler, ccdata) -> Optional[dict]:
             #    )
 
             # For now, only keep the last set of MO energies, even though it is
-            # printed at every step of geometry optimizations and fragment jobs.
+            # ed at every step of geometry optimizations and fragment jobs.
```
Comment (shivupa) [inline cclib/attribute_parsers/mosyms.py:189]: printed search and replace issue?

**Review by shivupa — APPROVED** [review]
LGTM after minor comment fix

File: cclib/attribute_parsers/scftargets.py [hunk 2]
```diff
@@ -61,6 +60,30 @@ def gaussian(file_handler, ccdata) -> Optional[dict]:
...
+                if "Energy threshold" in line:
+                    etarget = float(line.split()[-1])
+                if "Density threshold" in line:
+                    dtarget = float(line.split()[-1])
+                line = file_handler.virtual_next()
+
+            if getattr(ccdata, "scftargets") is None:
+                this_scftargets = []
+            else:
+                this_scftargers = ccdata.scftargets
```
Comment (berquist) [inline cclib/attribute_parsers/scftargets.py:79]:
```suggestion
                this_scftargets = ccdata.scftargets
```
Does this bug exist in the original code?

**Review by berquist — CHANGES_REQUESTED** [review]

**Comment by berquist** [comment]
Also good for rebase

File: cclib/attribute_parsers/scftargets.py (hunk 2 above)
Comment (amandadumi) [reply cclib/attribute_parsers/scftargets.py:79]: haha no i doubt it, probably just trying to go too quickly on my end.

Comment (amandadumi) [reply cclib/attribute_parsers/scftargets.py]: This is fixed now!

**Review by berquist — APPROVED** [review]
Sorry I missed that this was ready before I did the merge.  I'll rebase and merge.

**Comment by amandadumi** [comment]
Oh, thanks for doing that!!

**Pull request merged**